### ISSUE TITLE
Use new Docks feature

### ISSUE THREAD

**Issue by Scimonster** [pghilardi/atom-python-test#15]
As of [Atom 1.17](https://github.com/atom/atom/releases/tag/v1.17.0), there's a new Docks feature especially designed for panels like what this package opens. It would simplify usage from a user's point of view, and be more consistent with other Atom packages.

**Comment by pghilardi** [comment]
@Scimonster ,

You are totally right. The new docks feature is awesome and I will use them in this package. A new version with docks integrated will be released soon.

**Comment by dtenenba** [comment]
Would this enable the test panel to be resized? I would really like that...right now it takes up a lot of space on my small screen and I can't resize it. If not let me know and I will file a separate issue.
